src/network/fetchConnector.ts

```diff
@@ -0,0 +1,72 @@
+import fetch from "node-fetch";
```

Comment:
why do we have an axios connector and a fetch connector?

Reply:
The Fetch connector has been asked as an example on how easy it is to change the default connector (axios) if needed. It works the same way for the services.